code/backend/batch/utilities/helpers/DocumentProcessorHelper.py

```diff
@@ -22,20 +28,38 @@ def __init__(self):
         pass
 
     def process(self, source_url: str, processors: List[Processor]):
-        vector_store_helper = AzureSearchHelper()
-        vector_store = vector_store_helper.get_vector_store()
-        for processor in processors:
+        env_helper: EnvHelper = EnvHelper()
+        if env_helper.AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION:
```

Comment:
Could we have separate functions for whether IV is enabled or not?

Reply:
Updated